spec/controllers/questions_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe QuestionsController do
+  let(:user) {FactoryGirl.create :user}
+  let(:council){FactoryGirl.create :council}
+   let(:question) { FactoryGirl.create :question }
+
+   before do
+    stub_current_user user
+   end
+
+  describe "#index" do
+    it "Show all question to which belongs to user" do
+      get :index
+      expect(response).to be_success
```

Comment:
This is just checking the http status code - not that the action does what it's supposed to.